src/portal/src/app/repository/artifact-list-page/artifact-list-page.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+    <div class="breadcrumb" *ngIf="!withAdmiral">
+        <a (click)="goProBack()">{{'SIDE_NAV.PROJECTS'| translate}}</a>
+        <span class="back-icon"><</span>
+        <a (click)="watchGoBackEvt(projectId)">{{'REPOSITORY.REPOSITORIES'| translate}}</a>
+        <span *ngIf="referArtifactNameArray.length===1">&lt;<a (click)="backInitRepo()">{{repoName}}</a></span>
```

Comment:
Can this referArtifactNameArray be null or undefined?

Reply:
it has default  value  []